src/main/java/com/artipie/management/api/ApiAuthSlice.java

```diff
@@ -76,9 +78,9 @@ public Response response(
         final String line,
         final Iterable<Map.Entry<String, String>> headers,
         final Publisher<ByteBuffer> body) {
-        final Permission permission = new Permission.All(
+        final Permission permission = new Permission.Any(
```

Comment:
@olenagerasimova I am not sure that this change is correct. Take for example new user just registered from GitHub and trying to access dashboard. It won't be possible because `api` permission is missing and the only way to modify permissions is using dashboard. So I am pretty sure that one of permissions is enough, not both of them are required

Reply:
@olegmoz hm, that's why I've changed `All` to `Any`, or do I miss the statement?

Reply:
@olenagerasimova yes, I think we have misunderstanding. I am pretty sure that `Any` was right choice here. Again the problematic scenario is will occur at Artipie Central where users have no direct access to config files. New user that just came from GitHub with cookie won't be able to access dashboard, because he lacks `api` permission there. And this new user is not able to add this permission, as it is done via dashboard. This problem comes up if we switch to `All` here. In case of `Any` user (owner of the repository) can access dashboard using cookie. And other users could be granted `api` permission to access dashboard/API as well.

Reply:
@olegmoz 

> New user that just came from GitHub with cookie won't be able to access dashboard, because he lacks api permission there

`Any` allows access when least one of the origin permissions allows the action for a specified user, so in this case `ApiPermission` should allow action as user is managing his dashboard.

Reply:
@olenagerasimova I am so sorry! For some reason I understood that this PR changes `Any` to `All`, but it is other way around

Reply:
@olegmoz no problem, good we came to understanding)